src/lib/Components/Bidding/Screens/__tests__/SelectMaxBid-tests.tsx

```diff
@@ -1,3 +1,7 @@
+jest.mock("lib/Components/Bidding/Screens/ConfirmBid/PriceSummary", () => ({
```

Comment:
not clear to me why this diff is needed - can you explain?

Reply:
In test we generally don't mount any components that depend on `<QueryRenderer>` because they are just untestable (otherwise the error would be thrown).

```
Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

    Check the render method of `YourComponent`.
```

In `src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx`, we avoid this by using [an environment mock](https://github.com/artsy/emission/pull/1926/files#diff-e4b585673014c3d994c1ed23be87f302R129-R143). The price summary is perhaps not the thing we are interested in testing the most, so I think it makes sense to mock it out entirely.